hub/instruction_handling/call/generalities/account_rows_first_set.tex

```diff
@@ -87,7 +106,27 @@
 		we impose that
 		\begin{enumerate}
 			\item \If $\scenCallBalanceUpdateNotRequired _{i} = 1$ \Then $\accSameBalance      {i}{\callFirstCalleeAccountRowOffset}$
-			\item \If $\scenCallBalanceUpdateRequired _{i}    = 1$ \Then $\accIncrementBalance {i}{\callFirstCalleeAccountRowOffset}{\locValueLo}$
+			\item \If $\scenCallBalanceUpdateRequired    _{i} = 1$ \Then
+				\begin{enumerate}
+				        \item \If $\locIsCall = 1$ \Then
+						\[
+							\accIncrementBalance
+							{i}{\callFirstCalleeAccountRowOffset}{\locValueLo}
+						\]
+					\item \If $\locIsCallcode + \locIsDelegatecall + \locIsStaticcall = 1$ \Then
+						\[
+							\accSameBalance
+							{i}{\callFirstCalleeAccountRowOffset}
+						\]
+				\end{enumerate}
+				\saNote{}
+				Echoing
+				note~(\ref{hub: instruction handling: call: generalities: first account rows: value transfer subtleties}),
+				the above may alternatively be implemented without case disjunction as
+				\[
+					\accIncrementBalance
+					{i}{\callFirstCalleeAccountRowOffset}{\locIsCall \cdot \locValueLo}
+				\]
```

Comment:
What is actually implemented, see commit https://github.com/Consensys/linea-constraints/pull/512/commits/2087e3f94e4a4522334e0d05897def017d210a9d